lib/policy_machine/policy_element.rb

```diff
@@ -25,6 +25,11 @@ def connected?(other_pe)
       @pm_storage_adapter.connected?(self.stored_pe, other_pe.stored_pe)
     end
 
+    # Determines if self is a logical link to other node
+    def linked?(other_pe)
+      @pm_storage_adapter.linked?(self.stored_pe, other_pe.stored_pe)
```

Comment:
I was trying to be consistent with the rest of this file, and changing it everywhere would've required a lot of code changes unrelated to this PR. But alas, that ship has long since sailed. Changed everywhere in this file. All tests still pass:
Finished in 1 minute 29.08 seconds
1271 examples, 0 failures, 12 pending